packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -3,131 +3,62 @@
-import { CSSProperties, FC, createRef, useEffect, useState } from 'react';
+import { joinClassNames, radiusStyleMap, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import { FC } from 'react';
 import 'tailwindcss/tailwind.css';
 import { ICON_ASSET_ID } from './settings';
-import {
-    Alignment,
-    BlockSettings,
-    CalloutBlockProps,
-    CustomPaddingStyles,
-    Padding,
-    Type,
-    Width,
-    alignmentMap,
-    outerWidthMap,
-    paddingMap,
-    typeMap,
-} from './types';
-
-const getInnerDivClassName = (
-    type: Type,
-    width: Width,
-    alignment: Alignment,
-    isCustomPadding: boolean,
-    padding: Padding,
-    isCustomRadius: boolean,
-    cornerRadius: Radius
-): string =>
-    joinClassNames([
-        `tw-flex tw-items-center tw-text-white ${typeMap[type]}`,
-        width === Width.FullWidth && alignmentMap[alignment],
-        !isCustomPadding && padding && paddingMap[padding],
-        !isCustomRadius && cornerRadius && radiusStyleMap[cornerRadius],
-    ]);
-
-const getOuterDivClassName = (width: Width, alignment: Alignment): string =>
-    joinClassNames([outerWidthMap[width], width === Width.HugContents && alignmentMap[alignment]]);
+import { BlockSettings, CalloutBlockProps, Width, alignmentMap, outerWidthMap, paddingMap, typeMap } from './types';
 
 export const CalloutBlock: FC<CalloutBlockProps> = ({ appBridge }) => {
     const [blockSettings, setBlockSettings] = useBlockSettings<BlockSettings>(appBridge);
     const isEditing = useEditorState(appBridge);
     const { blockAssets } = useBlockAssets(appBridge);
     const { designTokens } = useGuidelineDesignTokens();
 
-    const {
-        type = Type.Warning,
-        alignment = Alignment.Left,
-        iconSwitch = true,
-        width = Width.FullWidth,
-        paddingChoice = Padding.M,
-        extendedRadiusChoice = Radius.None,
-        extendedRadiusTopLeft = '0px',
-        extendedRadiusTopRight = '0px',
-        extendedRadiusBottomLeft = '0px',
-        extendedRadiusBottomRight = '0px',
-        paddingTop = '0px',
-        paddingBottom = '0px',
-        paddingLeft = '0px',
-        paddingRight = '0px',
-        hasCustomPadding,
-        hasExtendedCustomRadius,
-        textValue,
-    } = blockSettings;
+    const outDivClassNames = joinClassNames([
+        outerWidthMap[blockSettings.width],
+        blockSettings.width === Width.HugContents && alignmentMap[blockSettings.alignment],
+    ]);
 
-    const [customPaddingStyle, setCustomPaddingStyle] = useState<CustomPaddingStyles>();
-    const [customCornerRadiusStyle, setCustomCornerRadiusStyle] = useState<CSSProperties>();
-    const [iconUrl, setIconUrl] = useState<string>();
-    const [iconAltText, setIconAltText] = useState<string>();
-    const blockRef = createRef<HTMLDivElement>();
+    const innerDivClassNames = joinClassNames([
```

Comment:
a more descriptive name could be helpful, since it's not clear what purpose of `innerDiv` and `outDiv` is.